04_environment/index.tex

```diff
@@ -160,26 +145,30 @@ \subsubsection{Incorporated Changes: Population Density}
 
-Another change that could be made, is to let islands choose who they can forage with. That could be in the form of direct messaging or voting. Essentially, our speculation is that agents would avoid the free-riding problem in both cases by enabling agents to make collective decisions based on communicated resource contributions. That’s because free-riders would be out-voted or not invited for foraging. That, in combination with the aforementioned tier system, would possibly result in all the islands contributing as much as possible to the input foraging “pool”.  However, it would be interesting to see if islands can then reach Pareto optimality, where every single agent gets the best possible return for their input.
+Another change that could be made is to let islands choose who they can forage with. That could be in the form of direct messaging or voting. Essentially, our speculation is that agents would avoid the free-riding problem in both cases by enabling agents to make collective decisions based on communicated resource contributions. That’s because free-riders would be out-voted or not invited for foraging. That, in combination with the aforementioned tier system, would possibly result in all the islands contributing as much as possible to the input foraging “pool”. However, it would be interesting to see if islands can then reach Pareto optimality, where every single agent gets the best possible return for their input.
 
-In addition to the above suggestions, area limited foraging where agents would only be allowed to forage with only one neighbouring island using a ranking algorithm. This could also be enforced and implemented in a future iteration. Moreover, this particular scenario could be extended by programming prey densities to be geofocused, where prey populations could vary between regions on the “map”. Islands would then need to make a strategic plan on how their ranking of neighbouring islands would change in accordance to their returns and returns broadcasted from other island pairs.
+In addition to the above suggestions, agents would be restricted to just a single foraging area where they would only be allowed to forage with one neighbouring island using a ranking algorithm. This could alter the game's dynamics because an agent would only have limited partners(islands) to forage with (other islands in the specified area) which could be explicitly decided or randomly allocated. This could also be enforced and implemented as a future work.Moreover, this particular scenario could be extended by programming prey densities to be geo-focused, where prey populations could vary between regions on the “map”. Islands would then need to make a strategic plan on how their ranking of neighbouring islands would change in accordance to their returns and returns broadcasted from other island pairs.
```

Comment:
limited partners (i.e. other islands). ....

his could also be enforced and implemented **as future work. Moreover,**

do NOT forget the space :)

revise the second sentence... I don't really understand why you include very long explanations in the parantheses :(